core/src/main/java/io/github/belgif/rest/guide/validator/output/model/ViolationFileObject.java

```diff
@@ -0,0 +1,11 @@
+package io.github.belgif.rest.guide.validator.output.model;
+
+public record ViolationFileObject(
```

Comment:
the "FileObject" part of the class name is a bit misleading. Maybe ViolationEntry